source/partials/next_release/_release.md.erb

```diff
@@ -1,73 +1,49 @@
 
-<h4>Bug Fixes</h4>
-* <%= link_to_commit 'cef6649637d4de2d7bedb6b00f0fa759cb310fe6', 'Fixes a race condition during Job reschedule', 3891 %>
-* <%= link_to_commit 'a1f99453ca629c18753a8835e406e621fa1cb560', 'Improves checksums for agent jar(s) download', 3884 %>
-* <%= link_to_commit '4db7e52bed39e539e4c6f71e789bee188e3e5bd2', 'Does not show the settings icon for config repo pipelines on VSM.', 3892 %> (<%= link_to_issue 3881%>)
-* <%= link_to_commit '686c76d62ad18ae8ffa1299bef2f82e3a225bb4c', 'Fixes issue with users authenticated using WebBasedAuthentication not assigned the right permissions', 3878 %>
 
 <h4>API Changes</h4>
-* <%= link_to_commit 'cba778c7770295fad4d036c82a7802caf41aa723', 'Introduces health check API for Server.', 3912 %>
-* <%= link_to_commit '97df2bfa2578349193c26250d2be59c0909dce58', 'Introduces health check api for agent', 3781 %> (<%= link_to_issue 3074%>)
-* <%= link_to_commit 'e4d041b2340ee6b7928a5fb11b6d3d787665dd1f', 'Adds v2 of Users API', 3720 %>
+* <%= link_to_commit 'd512703973e3ff1d9e1ea8fb5c3e94592504ec71', 'Fix config repo material representers to show only required fields', 3933%>
+* <%= link_to_commit 'd03b93f40082d82629bc6e2b30ddf86c3d86123e', 'Introduction of pipeline config API - v5' %> <%= link_to_issue 106 %>
 
 <h4>Docker Images</h4>
-* We have made a few changes to the <%= link_to("GoCD Server", "https://github.com/gocd/docker-gocd-server/blob/master/CHANGELOG.md#docker-gocd-server-17110")%> and the <%= link_to("GoCD Agent", "https://github.com/gocd/docker-gocd-agent/blob/master/CHANGELOG.md#docker-gocd-agent-17110")%> Docker images.
-
-<h4>Breaking Changes</h4>
-* Support for phusion based docker [server](https://hub.docker.com/r/gocd/gocd-server-deprecated/) and [agent](https://hub.docker.com/r/gocd/gocd-agent-deprecated/) images have been removed this release. We provide support for the docker images for GoCD agents across the following operating systems - Alpine [3.5](https://hub.docker.com/r/gocd/gocd-agent-alpine-3.5/), CentOS [6](https://hub.docker.com/r/gocd/gocd-agent-centos-6/) & [7](https://hub.docker.com/r/gocd/gocd-agent-centos-7/) , Debian [7](https://hub.docker.com/r/gocd/gocd-agent-debian-7/) & [8](https://hub.docker.com/r/gocd/gocd-agent-debian-8/), Ubuntu [12.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-12.04/), [14.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-14.04/) & [16.04](https://hub.docker.com/r/gocd/gocd-agent-ubuntu-16.04/)
 
 <h4>Deprecations</h4>
-* As of release 17.9, version 2 of the plugin info <%= link_to_api 'plugin-info' %> is deprecated. This version of the API will be removed in 17.12.0 release (scheduled to be released in November 2017). Version 3 of the API is available, and users are encouraged to use it.
+* <%= link_to_commit 'cdf6722bf17664799a81c42b5c2cc13ae48736fc', 'Remove the code related to starting vncserver', 4028 %> <%= link_to_issue 4005 %>
```

Comment:
This isn't a deprecation. The code is already removed. So, its a breaking change only to those users who are upgrading the agent bootstrappers.

Reply:
Also, add a link to this? https://github.com/gocd/gocd/issues/4005#issuecomment-348138015